### PR TITLE
Added @kesha-antonov/react-native-background-downloader

### PR DESCRIPTION
**Pull request by kesha-antonov**
Hi

# 📝 Why & how

Added library for background downloads: https://github.com/kesha-antonov/react-native-background-downloader

# ✅ Checklist

- [x] Added library to **`react-native-libraries.json`**

### CODE REVIEW

File: react-native-libraries.json [hunk 1]
```diff
@@ -10356,5 +10356,15 @@
...
+  },
+  {
+    "githubUrl": "https://github.com/kesha-antonov/react-native-background-downloader",
+    "npmPkg": "@kesha-antonov/react-native-background-downloader",
+    "examples": [
+      "https://github.com/kesha-antonov/react-native-background-downloader/tree/master/example"
+    ],
+    "ios": true,
+    "android": true,
+    "macos": true
```
Comment (Simek) [inline react-native-libraries.json:10368]: Hello @kesha-antonov, thank you for adding the new library! 👍

Entry looks good, however, we require an example when declaring out-of-tree platform support, like macOS, and example you have added do not include mac app. More information can be found in the Readme file section:
* https://github.com/react-native-community/directory?tab=readme-ov-file#%EF%B8%8F-out-of-tree-platforms

There are two ways to unblock this PR, either you can add macOS to the repository example, or the platform can be removed, and re-added in the future, when the example is updated or new one added.

Comment (kesha-antonov) [reply react-native-libraries.json]: Removed macos

**Review by Simek — APPROVED** [review]
Thank you for the update, LGTM! 👍

**Comment by kesha-antonov** [comment]
> Thank you for the update, LGTM! 👍 

Thanks

**Pull request merged**